core/src/main/java/io/questdb/cairo/DefaultCairoConfiguration.java

```diff
@@ -692,7 +692,7 @@ public int getParquetExportDataPageSize() {
 
     @Override
     public int getParquetExportRowGroupSize() {
-        return 0; // use default (512*512) rows
+        return 100_000;
```

Comment:
is this deliberate?

Reply:
Yes, it's better explicitly specify the default `rowgroupsize` value java code